src/less/donut-graph/donut-graph.less

```diff
@@ -0,0 +1,84 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+@import "../mixins/private/bubble-mixins.less";
+@import "../tooltip/tooltip.less";
+
+@inactive-opacity: 0.2;
+
+.donut-graph {
+    display: block;
+    width: 100%;
+}
+
+.donut-graph svg {
+    display: inline-block;
+    position: relative;
+    vertical-align: middle;
+    width: 100%;
+}
+
+.donut-graph__segment {
+    opacity: 1;
+    transition: opacity 0.3s;
+}
+.donut-graph__segment--primary {
+    .fill-token(donut-chart-primary, color-data-viz-chart-primary);
+    .stroke-token(donut-chart-primary, color-data-viz-chart-primary);
+}
+.donut-graph__segment--secondary {
+    .fill-token(donut-chart-secondary, color-data-viz-chart-secondary);
+    .stroke-token(donut-chart-secondary, color-data-viz-chart-secondary);
+}
+.donut-graph__segment--tertiary {
+    fill: url("#dataVizPattern0");
+    .stroke-token(donut-chart-quaternary, color-data-viz-chart-quaternary-stroke);
+}
+.donut-graph__segment--quaternary {
+    fill: url("#dataVizPattern1");
+    .stroke-token(donut-chart-tertiary, color-data-viz-chart-tertiary-stroke);
+}
+.donut-graph__segment--quinary {
+    .fill-token(donut-chart-quinary-background, color-data-viz-chart-quinary-background);
+    .stroke-token(donut-chart-quinary-stroke, color-data-viz-chart-quinary-stroke);
+}
+.donut-graph--inactive {
+    opacity: @inactive-opacity;
+}
+
+.donut-graph__tooltip {
```

Comment:
Is this needed if we are using tooltip?

Reply:
@agliga yes this is needed, not all of the code was reusable, as it caused double display and double position values in the compiled code which fails the linter.  I also had to prevent pointer-events since this tooltip follows the mouse and it caused a flicker when it accepted pointer events.